src/Algebra/Graph/AdjacencyMap/Algorithm.hs

```diff
@@ -53,6 +56,7 @@ import qualified Data.Set                            as Set
 --                                          , Node { rootLabel = 3
 --                                                 , subForest = [ Node { rootLabel = 4
 --                                                                      , subForest = [] }]}]
+-- 'forest' (dfsForest ('circuit' [1..5] + 'transpose' ('circuit' [1..5]))) == 'path' [1..5]
```

Comment:
This example appears to be untested.

Reply:
Ah, I was going to contrast the dfs-order to the similar example on bfs. Should I add a test or delete the line here?

Reply:
It's a useful example, but I'd like all the examples to be tested. Please add the corresponding test!